channels/sync_runner_test.go

```diff
@@ -46,6 +46,21 @@ func TestRequireAccess(t *testing.T) {
 	assertRejected(t, result, base.HTTPErrorf(403, "missing channel access"))
 }
 
+func TestRequireAdmin(t *testing.T) {
+	const funcSource = `function(doc, oldDoc) { requireAdmin() }`
+	runner, err := NewSyncRunner(funcSource)
+	assert.Equals(t, err, nil)
+	var result interface{}
+	result, _ = runner.Call(parse(`{}`), parse(`{}`), parse(`{}`))
```

Comment:
AFAIK, this is equivalent to how a request from the Admin port will hit the Sync Function, but would appreciate a double-check on this.